ops/testing.py

```diff
@@ -990,9 +990,11 @@ def add_relation_unit(self, relation_id: int, remote_unit_name: str) -> None:
         app = relation.app
         if not remote_unit_name.startswith(app.name):
             warnings.warn(
-                'Remote unit name invalid: the remote application of {} is called {!r}; '
-                'the remote unit name should be {}/<some-number>, not {!r}.'
-                ''.format(relation_name, app.name, app.name, remote_unit_name))
+                'Remote unit name invalid: '
```

Comment:
As above, I think the ruff formatter wants the leading bit consistent.

```suggestion
                f'Remote unit name invalid: '
```